server/schedulers/balance_test.go

```diff
@@ -304,6 +305,34 @@ func (s *testBalanceLeaderSchedulerSuite) TestScheduleWithOpInfluence(c *C) {
 	c.Check(s.schedule(), IsNil)
 }
 
+func (s *testBalanceLeaderSchedulerSuite) TestTransferLeaderOut(c *C) {
+	// Stores:     1    2    3    4
+	// Leaders:    7    8    9   12
+	s.tc.AddLeaderStore(1, 7)
+	s.tc.AddLeaderStore(2, 8)
+	s.tc.AddLeaderStore(3, 9)
+	s.tc.AddLeaderStore(4, 12)
+	s.opt.TolerantSizeRatio = 0.1
+	for i := uint64(1); i <= 7; i++ {
+		s.tc.AddLeaderRegion(i, 4, 1, 2, 3)
+	}
+
+	// balance leader: 4->1, 4->1, 4->2
```

Comment:
can we check if the target ids are correct?

Reply:
There is no function to get target store id in `op`?

Reply:
`op.Step(0).ToStore`

Reply:
Thanks